### DISCUSSION TITLE
Need assistance. Upgrading to Laravel 9 from 8

### DISCUSSION THREAD

**Discussion by michaelgavan** [laravel/framework#43134] [Q&A] [answered]
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - illuminate/support[v8.0.0, ..., v8.11.2] require php ^7.3 -> your php version (8.1.7) does not satisfy that requirement.
    - Root composer.json requires infyomlabs/laravel-generator 8.0.x-dev -> satisfiable by infyomlabs/laravel-generator[8.0.x-dev].
    - Conclusion: don't install laravel/framework v9.19.0 (conflict analysis result)
    - infyomlabs/laravel-generator 8.0.x-dev requires illuminate/support ^8.0 -> satisfiable by illuminate/support[v8.0.0, ..., 8.x-dev].
    - Only one of these can be installed: illuminate/support[dev-master, v1.0.0, v1.1.0, v1.1.1, v1.1.2, v4.0.0-BETA2, ..., 4.2.x-dev, v5.0.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev, 10.x-dev], laravel/framework[v9.19.0, 9.x-dev]. laravel/framework replaces illuminate/support and thus cannot coexist with it.
    - Root composer.json requires laravel/framework ^9.19 -> satisfiable by laravel/framework[v9.19.0, 9.x-dev].

**Comment by yanalshoubaki** [comment, marked as answer]
At first, delete vendor folder.
then change laravel/framework version to ^9.19, and change other packages versions to the version that support laravel 9.

I think this will help you.

**Comment by rodrigopedra** [comment]
> Laravel's GitHub issue trackers are not intended to provide Laravel help or support.

read more at the docs: https://laravel.com/docs/9.x/contributions#support-questions

**Comment by erikn69** [comment]
Also https://laravel.com/docs/9.x/upgrade
